.env.example

```diff
@@ -1,3 +1,5 @@
+# API_THROTTLE=60,1,api
```

Comment:
this can be defined by any number of route prefixes?

Reply:
what do you mean?

Reply:
just interested in how this spec works. might be worth linking the laravel doc for it?

ie. can we specify multiple throttling rules in a single line>

Reply:
No, the arguments are `rate limit`, `reset interval`, `key prefix`. The prefix is there because if there is an existing matching timer key in redis, it will use the expiry on that one. Multiple rules on the same line will need a different syntax.

Reply:
The use case here is specifically for the replay retrieval endpoint. Is support for that planned to be added at a later date? Might be worth considering it before merging this.